tests/core/instruction/gates/test_custom_gate.py

```diff
@@ -10,8 +11,8 @@
     ATOSDevice,
     AvailableDevice,
     AWSDevice,
-    IBMDevice,
     GOOGLEDevice,
+    IBMDevice,
```

Comment:
why moving IBM after ? was it doen by isort ?

Reply:
Yes.